lib.js

```diff
@@ -140,7 +140,7 @@ export async function filesFromPaths (paths, options) {
       files.push(file)
       const nameParts = file.name.split('/')
```

Comment:
i think maybe u also want to let windows play your wild path prefix matching game.
```suggestion
      const nameParts = file.name.split(path.sep)
```
https://nodejs.org/docs/latest-v18.x/api/path.html#pathsep